server/modules/filter/luafilter.c

```diff
@@ -105,7 +105,7 @@ static FILTER_OBJECT MyObject =
  * The module entry point routine. It is this routine that
  * must populate the structure that is referred to as the
  * "module object", this is a structure with the set of
- * external entry points for this module.
+ * external entry ponits for this module.
```

Comment:
Although fixing typos is a good thing, they should be done as a separate change in a different pull request if they are done. I think we can leave the typos in other modules be for now.